### PR TITLE
fix: Fix the dependencies

### PR DESCRIPTION
**Pull request by joscha**
PTAL @jpnelson

### CODE REVIEW

File: packages/lerna-semantic-release-analyze-commits/package.json [hunk 1]
```diff
@@ -5,12 +5,9 @@
     "url": "https://github.com/atlassian/lerna-semantic-release/issues"
   },
   "dependencies": {
+    "@semantic-release/commit-analyzer": "~2.0.0",
     "lerna-semantic-release-utils": "^4.0.0"
   },
-  "peerDependencies": {
-    "semantic-release": "semantic-release/semantic-release#a5cb9ea7f7a66657f95f0d913126e812adb6d9f8",
-    "@semantic-release/commit-analyzer": "~2.0.0"
```
Comment (joscha) [inline packages/lerna-semantic-release-analyze-commits/package.json:12]: the analyzer was only used in this package, so doesn't need to be a peer. semantic-release itself was not used at all.

File: packages/lerna-semantic-release-get-last-release/package.json [hunk 2]
```diff
@@ -5,12 +5,7 @@
     "url": "https://github.com/atlassian/lerna-semantic-release/issues"
   },
   "dependencies": {
-    "lerna-semantic-release-utils": "^4.0.0",
-    "lerna-semantic-release-io": "^5.0.0"
-  },
-  "peerDependencies": {
```
Comment (joscha) [inline packages/lerna-semantic-release-get-last-release/package.json:11]: wasn't used

File: packages/lerna-semantic-release-get-last-release/package.json [hunk 3]
```diff
@@ -5,12 +5,7 @@
     "url": "https://github.com/atlassian/lerna-semantic-release/issues"
   },
   "dependencies": {
-    "lerna-semantic-release-utils": "^4.0.0",
-    "lerna-semantic-release-io": "^5.0.0"
```
Comment (joscha) [inline packages/lerna-semantic-release-get-last-release/package.json:9]: io wasn't used

File: packages/lerna-semantic-release/package.json [hunk 4]
```diff
@@ -39,7 +39,6 @@
     "lerna"
   ],
   "license": "MIT",
-  "main": "bin/semantic-release.js",
```
Comment (joscha) [inline packages/lerna-semantic-release/package.json:42]: this file does not exist

**Comment by rmanalan** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;QwfnmYEK&#x2F;12-some-stuff"> some stuff</a>

**Pull request merged**